demos/atk-init.php

```diff
@@ -42,19 +42,21 @@
 $app->initLayout($app->stickyGET('layout') ?: 'Maestro');
 
 $layout = $app->layout;
-// Need for phpUnit test only for producing right url.
+// Need for phpunit only for producing right url.
 $layout->name = 'atk_admin';
 $layout->id = $layout->name;
 
 if ($layout instanceof \atk4\ui\Layout\Navigable) {
-    $layout->addMenuItem(['Welcome to Agile Toolkit', 'icon' => 'gift'], ['/demos/index']);
+    $demosUrl = preg_replace('~(?<=/demos/).+~s', '', $_SERVER['REQUEST_URI']);
+
```

Comment:
Why do you need 'REQUEST_URI' here? The value '/demos/' is only for setting proper link to menu item. There is no need fro pre_replace.

Reply:
Consider having /demos/ in non web root and/or beiing in a demo in subdir.

Reply:
@ibelar so ok to merge?